dist/index.js

```diff
@@ -7042,6 +7046,17 @@ module.exports.enforce = async function() {
     }
 };
 
+function getLabels(skipLabel, skipLabels) {
+    if (skipLabel != '') {
+        return [skipLabel]
+    }
+    return labelExtractor.getLabels(skipLabels)
```

Comment:
Should you check if `skipLabel` and `skipLabels` are defined at the same time and error or warn?

Reply:
Great idea. I'll update this to add a warning message to the action if the `skipLabel` input is being used. 👍🏼

Reply:
Despite `skipLabel` being deprecated it is still being prioritized. That by design? Maybe another warning when both fields are used at the same time? Or even an error making them mutually exclusive.

Reply:
I see what you are saying. 

My initial thought was that this makes the path to upgrade more straightforward. When a workflow explicitly uses the `skipLabels` input variable or just uses defaults, there is nothing to do. If a workflow that already exists is using `skipLabel`, then that value _should_ be prioritized initially and ask the user to use the new input variable.

Reply:
Agreed, migration/upgrading should be easy. I don't think the additional mutual exclusion check would stand in the way
```javascript
if (skipLabel != '' && skipLabels != '') { 
  error 
}
```
 For those who just upgrade, they'll only have `skipLabel`, so only the warning about using the other label. The error is there for those who want to migrate to new label, the error prevents them from shooting themselves in the foot by having both labels, but only the former being used.

Reply:
Ah I gotcha. Makes sense. TY!

Reply:
So thought about this a little more. And I think that I still prefer not throwing an error and prioritizing the old input variable for a time. 

My reasoning is that if the enforcer were to check for both input variables being set, every existing workflow with an explicit value for `skipLabel` being break. This is because the default for `skipLabels` is set so that the `Skip-Changelog` label works "out of the box". Both input variables would then be set, and it is likely the old input variable is a different value and should be prioritized during a migration. And I don't want to do a hard-breaking change - I'd prefer to deprecate the var and in a future release remove the variable.

Is that making sense?

Reply:
I gotcha. Didn't think about the defaults for `skipLabels`. And you don't want to omit the default because that would make it harder for newcomers to this action to get off the ground. The warning should give enough guidance.